src/hipscat/io/file_io/file_pointer.py

```diff
@@ -67,20 +70,19 @@ def get_file_pointer_for_fs(protocol: str, file_pointer: FilePointer) -> FilePoi
             split_pointer = file_pointer.split("file://")[1]
         else:
             split_pointer = file_pointer
-    if protocol == "abfs":
-        #return the path minus protocol+account name
-        split_pointer = file_pointer.split("abfs://")[1]
-    if protocol == "s3":
-        #just strip the protocol, and keep the bucket name
-        split_pointer = file_pointer.split("s3://")[1]
+    elif protocol in ["abfs", "s3"]:
+        #return the path minus protocol://
+        split_pointer = file_pointer.split(f"{protocol}://")[1]
+    else:
```

Comment:
This exception makes me nervous - we should probably just pass-through any unknown file system instead of limiting ourselves to these 3 only.

Reply:
I agree, it'd be nice to have more protocols and it seems like others might work with the same code

Reply:
Ok, I've removed all implementations of "supported protocos" and rely on fsspec to notify the users if there are issues with the filesystem generations.